src/publish/inputs.js

```diff
@@ -6,6 +6,11 @@ exports.default = async function inputs({ context }) {
     : "";
   const path = "." + (titleMatch.path || "");
 
+  const mergeTargetParser = /^Merge target: (?<merge_target>.+)$/m;
+  let { merge_target } = context.payload.issue.body.match(
+    mergeTargetParser
+  ).groups;
```

Comment:
True, forgot about the fallback for destructuring. Because other than this, `merge_target` can be undefined, as it should evaluate to empty env variable, thus being skipped in cli call. (not that I haven't test any of this yet, as its not decided what we'll do with the original issue just yet).